Subject: verdin-imx8mm: add fdtoverlay load address

so the extlinux parser can correctly load and apply overlays.

Signed-off-by: Lucas Hehl <lucas@hehl.com.br>
---
 include/configs/verdin-imx8mm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/verdin-imx8mm.h b/include/configs/verdin-imx8mm.h
index c54383b886eb8e68bccde4e08bd3e8529d6e15c1..e45e98e571360ed59ee243d4db824bcd7f63737e 100644
--- a/include/configs/verdin-imx8mm.h
+++ b/include/configs/verdin-imx8mm.h
@@ -20,6 +20,7 @@
 
 #define MEM_LAYOUT_ENV_SETTINGS \
 	"fdt_addr_r=0x50200000\0" \
+	"fdtoverlay_addr_r=0x53000000\0" \
 	"kernel_addr_r=" __stringify(CONFIG_SYS_LOAD_ADDR) "\0" \
 	"kernel_comp_addr_r=0x40200000\0" \
 	"kernel_comp_size=0x08000000\0" \
